pkgs/tools/typesetting/tex/tex-match/default.nix

```diff
@@ -0,0 +1,31 @@
+{ rustPlatform, fetchFromGitHub, gtk3, pkg-config, glib, lib }:
+
+let
+  version = "1.2.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "tex-match";
+  inherit version;
```

Comment:
```suggestion
rustPlatform.buildRustPackage {
  pname = "tex-match";
  version = "1.2.0";
```

Reply:
Missing a `rec` there. `let`'d `version` is fine IMO.

Reply:
> Missing a `rec` there. `let`'d `version` is fine IMO.

It's a matter of personal taste, but indeed `rec` is missing.